homeassistant/helpers/config_validation.py

```diff
@@ -255,6 +256,17 @@ def time_zone(value):
 weekdays = vol.All(ensure_list, [vol.In(WEEKDAYS)])
 
 
+def url(value: Any) -> str:
```

Comment:
I think it's fine as `url`. The definition of URL is that it is a universal resource locator, not something specifically made for HTTP(S) only. And it keeps our existing syntax of `cv.string`, `cv.bool`, etc.